src/Command/Generate/PluginConditionCommand.php

```diff
@@ -97,9 +97,9 @@ protected function interact(InputInterface $input, OutputInterface $output)
     {
         $io = new DrupalStyle($input, $output);
 
-        $entity_manager = $this->getEntityManager();
+        $entityTypeRepository = $this->getService('entity_type.repository');
```

Comment:
@naveenvalecha At some point (probably no anytime soon) we are going to stop adding service at the ContainerAware because it can be a mess to maintain.

We  do not have a stable release yet, so we are not maintaining any backward compatibility yet.